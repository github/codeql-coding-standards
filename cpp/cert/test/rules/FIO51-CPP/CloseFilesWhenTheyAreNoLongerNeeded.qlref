rules/FIO51-CPP/CloseFilesWhenTheyAreNoLongerNeeded.ql