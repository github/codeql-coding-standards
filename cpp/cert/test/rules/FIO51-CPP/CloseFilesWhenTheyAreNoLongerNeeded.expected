| test.cpp:6:3:6:16 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:5:21:5:27 | call to basic_fstream | call to basic_fstream |
| test.cpp:63:3:63:16 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:58:10:58:13 | call to open | call to open |
| test.cpp:77:3:77:16 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:70:9:70:9 | call to operator<< | call to operator<< |
| test.cpp:91:5:91:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:89:8:89:8 | call to operator>> | call to operator>> |
| test.cpp:94:5:94:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:92:8:92:13 | call to gcount | call to gcount |
| test.cpp:97:5:97:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:95:8:95:10 | call to get | call to get |
| test.cpp:100:5:100:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:98:8:98:14 | call to getline | call to getline |
| test.cpp:103:5:103:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:101:8:101:13 | call to ignore | call to ignore |
| test.cpp:106:5:106:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:104:8:104:11 | call to peek | call to peek |
| test.cpp:109:5:109:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:107:8:107:11 | call to read | call to read |
| test.cpp:112:5:112:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:110:8:110:15 | call to readsome | call to readsome |
| test.cpp:115:5:115:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:113:8:113:14 | call to putback | call to putback |
| test.cpp:118:5:118:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:116:8:116:12 | call to unget | call to unget |
| test.cpp:121:5:121:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:119:8:119:12 | call to tellg | call to tellg |
| test.cpp:124:5:124:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:122:8:122:12 | call to seekg | call to seekg |
| test.cpp:127:5:127:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:125:8:125:11 | call to sync | call to sync |
| test.cpp:130:5:130:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:128:8:128:8 | call to operator<< | call to operator<< |
| test.cpp:133:5:133:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:131:8:131:10 | call to put | call to put |
| test.cpp:136:5:136:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:134:8:134:12 | call to write | call to write |
| test.cpp:139:5:139:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:137:8:137:12 | call to tellp | call to tellp |
| test.cpp:142:5:142:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:140:8:140:12 | call to seekp | call to seekp |
| test.cpp:145:5:145:18 | call to terminate | The execution terminates without closing the `fstream` last accessed in $@. | test.cpp:143:8:143:12 | call to flush | call to flush |
